sequencer/src/api/sql.rs

```diff
@@ -442,8 +442,8 @@ where
     .bind(param)
     .fetch_one(tx.as_mut())
     .await?;
-    let proposal: Proposal<SeqTypes, QuorumProposal<SeqTypes>> = bincode::deserialize(&data)?;
-    Ok(Leaf::from_quorum_proposal(&proposal.data))
+    let proposal: Proposal<SeqTypes, QuorumProposal2<SeqTypes>> = bincode::deserialize(&data)?;
```

Comment:
are we inserting QuorumProposal1 or QuorumProposal2 in database? If we are storing QuorumProposal2 then we should have a migration to make the existing proposals (q1) to q2?